packages/now-node/test/fixtures/07-content-length/vercel.json

```diff
@@ -1,8 +0,0 @@
-{
-  "version": 2,
-  "builds": [
-    { "src": "test1.js", "use": "@vercel/node" },
-    { "src": "test2.js", "use": "@vercel/node" },
-    { "src": "test3.js", "use": "@vercel/node" }
-  ]
-}
```

Comment:
This file should not have been deleted. I think the merge conflict was solved incorrectly. Take a look at PR #5925

Reply:
Some files in this directory get me a little bit confused since I am guessing they are generated by the tests. In master there was a test3.js but not test1 and test2 which appeared in my branch only. Not sure why vercel.json didn't get renamed correctly under the node (not now-node) path either.

Reply:
File is back now

Reply:
No its not generated, take a look at #5925 and you'll see some files were deleted and some were renamed.

Reply:
yes, vercel.json files were deleted by mistake but still not sure what's going on with the testX.js files